src/compiler/emitter.ts

```diff
@@ -7312,13 +7340,32 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
                 write("});");
             }
 
+            function lookupSpecifierName(declaration: ImportEqualsDeclaration | ImportDeclaration | ExportDeclaration): string {
+                let specifier: Node;
+                if (declaration.kind === SyntaxKind.ImportEqualsDeclaration) {
+                    specifier = (declaration as ImportEqualsDeclaration).moduleReference;
+                }
+                else {
+                    specifier = (declaration as ImportDeclaration | ExportDeclaration).moduleSpecifier;
+                }
+                let moduleSymbol = resolver.getSymbolAtLocation(specifier);

```

Comment:
this code looks very similar to `emitExternalModuleSpecifier` in declaration emitter. I'm curious if it can be moved to checker so resolver will return the final result